src/explore-education-statistics-frontend/src/modules/data-catalogue/DataSetFilePage.tsx

```diff
@@ -104,10 +104,14 @@ export default function DataSetFilePage({ dataSetFileId }: Props) {
   }
 
   const {
-    file,
+    file: {
+      id: fileId,
+      meta: { timePeriod, filters, geographicLevels, indicators },
+      // dataCsvPreview: { headers, rows },
+      // variables: variables,
```

Comment:
Can these properties be removed?

Reply:
I added those to be helpful for whoever adds this stuff to the frontend